regression/goto-gcc/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+string(REGEX REPLACE "goto-cc$" "goto-gcc"
+  goto_gcc_path "$<TARGET_FILE:goto-cc>")
```

Comment:
Seems reasonable. Consider adding a comment explaining why this is necessary, though.

Reply:
Thanks, done - hoping it's clear/useful?

Reply:
I think so :+1: